admin-ui/app/locales/fr/translation.json

```diff
@@ -56,7 +56,7 @@
     "aliases": "Aliases",
     "trust_relationships": "Relations de confiance",
     "api_config": "Configurer les propriétés de l'API",
-    "builtIn": "Intégré",
+    "basic": "Basique",
```

Comment:
Replace remaining `builtIn` key in Spanish translation

- In admin-ui/app/locales/es/translation.json (line 225), rename  
  `"builtIn": "Integrado"`  
  to  
  `"basic": "Integrado"`

No other `menus.builtIn` references found.